.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+## Changes description
+<!--- Describe your changes in details. -->
+
+## Context
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an opened issue, please link to the issue here. -->
+
+## Checklist
+<!--- Feel free to add other steps if needed. -->
+
+- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Please, don't request your main!
+- [ ] Check commits & PR names matches our requested structure. It must follow the https://www.conventionalcommits.org pattern.
+- [ ] Check your code additions will fail neither code linting checks.
+- [ ] I have reviewed the submitted code.
+- [ ] I have tested on a phone device/emulator.
+- [ ] I have tested on a tablet device/emulator.
+- [ ] If it's a new component, please ask for design review.
+
+## Does this introduce a breaking change?
+<!-- If this introduces a breaking change, please describe the impact and migration path for existing applications below. -->
+
+- Yes
+- No
+
+## Screenshots
+
+#### Phone
+<!--- Put your phone screenshots here. -->
+
+#### Tablet
+<!--- Put your tablet screenshots here. -->
+
+#### Large screen
```

Comment:
Not sure this is relevant for iOS...
It could be but I think we then have to precise what we mean by large screen : iPad Prod 12.9" ?

Reply:
Agreed with you, it's maybe not relevant here.

Reply:
you're right ;)